README.md

```diff
@@ -1123,16 +1123,28 @@ npm run lint
   const goodNewArray = [...originalArray];
   ```
 
-- [8.5](#8.5) <a name="8.5"></a> To convert from an array-like object to an array (for example, a `NodeList` returned by `document.querySelectorAll`, or a jQuery object), use `Array.from`.
+- [8.5](#8.5) <a name="8.5"></a> To convert from an array-like object to an array use `Array.from`.
```

Comment:
I'm in favor of ditching this, although spread throws in this example because it has no iteration protocol, e.g.

```js
const arrayLikeObject = {
  0: 'a',
  1: 'b',
  length: 2,

  [Symbol.iterator]() {
    let index = 0;

    return {
      next: () => {
        return this[index]
          ? {value: this[index++], done: false} 
          : {done: true};
      },
    };
  },
};
```